lib/screens/feed.dart

```diff
@@ -1,4 +1,6 @@
 import 'package:flutter/material.dart';
+import 'package:untitled/widgets/photo_container.dart';
```

Comment:
Could you add `package:flutter/widgets.dart` to export other files in the `widgets` directory? so we can just import the file